tests/cli/test_datasets.py

```diff
@@ -475,6 +496,10 @@ def test_add_and_create_dataset(
     assert 1 == result.exit_code
     assert 'Dataset "new-dataset" does not exist.' in result.output
 
+    existing_file = client.path / datadir / "myfile"
```

Comment:
Maybe put `myfile` in a sub-dir to test that directories aren't added.

Reply:
You mean empty directories? Or why not add subdirectories?

Reply:
I mean `existing_file = client.path / datadir / sub-dir / "myfile"`. Then checking that `datadir / sub-dir` is not added to dataset's metadata. This test should fail now because of using `rglob`.

Reply:
ok. That should be done (the test checks for files so if there was a folder as well, there'd be more than 4 paths returned